apps/meteor/client/sidebar/header/EditStatusModal.tsx

```diff
@@ -46,6 +46,21 @@ const EditStatusModal = ({ onClose, userStatus, userStatusText }: EditStatusModa
 		onClose();
 	}, [dispatchToastMessage, statusType, statusText, setUserStatus, onClose, t]);
 
+	const inputRef = useRef(null);
+	const handleEmojiPicker = (event: any): void => {
+		event.stopPropagation();
+		event.preventDefault();
+		if (EmojiPicker.isOpened()) {
+			EmojiPicker.close();
+		}
+		EmojiPicker.open(inputRef.current, (emoji: any) => {
+			const emojiValue = `:${emoji}:`;
+			inputRef.current.focus();
+			if (!document.execCommand || !document.execCommand('insertText', false, emojiValue)) {
```

Comment:
`execCommand` is deprecated. Don't know enough about the EmojiPicker code at this moment so not going to advice anything.